src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDateFormat sdf = new SimpleDateFormat(DATE_PATTERN);
```

Comment:
If you create a formatter, make it a constant field. This will make your code easier to understand and maintain. Declare SimpleDateFormat as a static final field and use it throughout your class.